src/OrchardCore.Modules/OrchardCore.Search.Lucene/Services/LuceneContentPickerResultProvider.cs

```diff
@@ -28,15 +28,8 @@ public LuceneContentPickerResultProvider(
 
     public async Task<IEnumerable<ContentPickerResult>> Search(ContentPickerSearchContext searchContext)
     {
-        var indexName = "Search";
-
         var fieldSettings = searchContext.PartFieldDefinition?.GetSettings<ContentPickerFieldLuceneEditorSettings>();
 
-        if (!string.IsNullOrWhiteSpace(fieldSettings?.Index))
-        {
-            indexName = fieldSettings.Index;
-        }
```

Comment:
Maybe not using `indexName` is actually a bug? Looks like it should have defaulted to "Search"...